### PR TITLE
#1399. [Records] Check a big record with a large number of positional fields

### PR DESCRIPTION
**Pull request by sgrekhov**

### CODE REVIEW

File: LanguageFeatures/Records/members_A04_t04.dart [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A record type declares all of the members defined on [Object]. It
+/// also exposes getters for each named field where the name of the getter is
+/// the field's name and the getter's type is the field's type. For each
+/// positional field, it exposes a getter whose name is $ followed by the number
+/// of preceding positional fields and whose type is the type of the field.
+///
+/// @description Checks record with the big number of positional fields
```
Comment (eernstg) [inline LanguageFeatures/Records/members_A04_t04.dart:11]: Perhaps this flows more naturally:
```suggestion
/// @description Checks record with a large number of positional fields
```

**Review by eernstg — COMMENTED** [review]
Just one small suggestion.

**Comment by sgrekhov** [comment]
@eernstg please approve

**Review by eernstg — APPROVED** [review]
LGTM

**Pull request merged**